examples/front-proxy/service-envoy.yaml

```diff
@@ -19,7 +19,11 @@ static_resources:
               - "*"
               routes:
               - match:
-                  prefix: "/service"
+                  prefix: "/plain"
+                route:
```

Comment:
i dont think you need this - just use a single route with 2 upstream file paths - it detracts from the point that compression happens because of the `content-type`

Reply:
also copy this file and edit in the gzip folder - not in the front-proxy folder - its already confused/confusing enough 

the intention is to completely separate the examples from the front-proxy folder - i just havent found time to do it yet

Reply:
My mistake